src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -289,7 +289,7 @@ export class PageControllerBase {
       const context = model.getFormContext(request, state)
 
       // Redirect back to last relevant page
-      if (!context.paths.includes(path)) {
+      if (!context.force && !context.paths.includes(path)) {
```

Comment:
Morning @davidjamesstone

Just realised that putting the override here stops us using ~`progress`~ `paths` in future

E.g. Users would be granted forced access but the back link wouldn't work correctly

Instead, can we suppress the [loop break in `getFormContext()`](https://github.com/DEFRA/forms-runner/blob/conditions-exit-page-validate-force/src/server/plugins/engine/models/FormModel.ts#L300-L303) instead?

**Before**
We stop pushing to `paths` when we hit errors
```js
// Stop at current page or with errors
if (path === currentPath || collection.getErrors(errors)) {
```

**After**
We keep pushing to `paths` when we hit errors but for form previews only
```js
// Stop at current page or with errors
if (path === currentPath || (collection.getErrors(errors) && !isPreview)) {
```

Probably don't even need to stop at the current page because it's [in the loop above](https://github.com/DEFRA/forms-runner/blob/conditions-exit-page-validate-force/src/server/plugins/engine/models/FormModel.ts#L273-L276) already